classes/class-admin.php

```diff
@@ -25,6 +25,11 @@ class Admin {
 	public $live_update;
 
 	/**
+	 * @var Export
+	 */
+	public $export;
```

Comment:
It is possible. One option would be an adapter class between the two, or moving several of the methods we just migrated away from Admin back into admin.

> On Feb 29, 2016, at 5:26 PM, Luke Carbis <notifications@github.com> wrote:
> 
> In classes/class-admin.php <https://github.com/xwp/stream/pull/823#discussion_r54509913>:
> 
> > @@ -25,6 +25,11 @@ class Admin {
> >  	public $live_update;
> >  
> >  	/**
> > +	 * @var Export
> > +	 */
> > +	public $export;
> Is it possible to decouple Exports from List_Table?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/xwp/stream/pull/823/files#r54509913>.
>